app/controllers/passwordless/sessions_controller.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+require "bcrypt"
+
+module Passwordless
+  # Controller for managing Passwordless sessions
+  class SessionsController < ApplicationController
```

Comment:
Why has this file been added? I was under the impression it's imported automatically by the gem and we can keep it out of the application code?

Reply:
we need to modify the render call slightly to get it to call the right layout